build.gradle

```diff
@@ -99,8 +99,9 @@ allprojects {
             versionSeparator = ''
             versionIncrementer 'incrementMinor'
         }
+        //override the version
+        versionCreator({versionFromTag,scmPosition -> "5.3.0-alpha"})
```

Comment:
Thanks @munir-rakuten .  This can be removed when merging to master. I will add this in documentation.